ext/cbc-wrapper/cbc_wrap.c

```diff
@@ -1851,8 +1851,8 @@ struct timeval rb_time_timeval(VALUE);
 #endif
 
 
-#include "Coin_C_defines.h"
-#include "Cbc_C_Interface.h"
+#include "coin/Coin_C_defines.h"
```

Comment:
This file is automatically generated by `swig`. These 2 lines are defined in `cbc_wrap.i` that `swig` uses as its input. It would probably be good to update the `cbc_wrap.i` file too.
I don't think it is necessary to rerun swig in this case though.

Reply:
Updated! I have updated `cbc.i` and confirmed this line is generated by `swig` command (with many extra diffs because my `swig` is 3.0.12.)